File: 2.0/python/texts/findTextsNonProfitAgency.py
from callfire.client import CallfireClient

client = CallfireClient('api-login', 'api-password')
response = client.texts.findTexts(
    inbound=True,
    campaignId=1003002,
    limit=10
).result()

# see sample JSON response for this API
# on 'curl' samples tab
print(response)

File: 2.0/php/texts/findTextsNonProfitAgency.php
<?php

class ApiClientSample {

    public static function main() {
        $client = \CallFire\Api\DocumentedClient::createClient("login", "password");
        $request = $client->findTexts();
        $request->getOperationConfig()->setQueryParameters(array("campaignId" => 1003002,
                                                                 "inbound" => true,
                                                                 "limit" => 10));
        $result = $client->request($request);
        $json = json_decode($result->getBody());
    }
}

ApiClientSample::main();
